codalab/lib/bundle_cli.py

```diff
@@ -2065,12 +2069,23 @@ def do_wadd_command(self, args):
         ),
     )
     def do_gls_command(self, args):
-        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec)
-        group_dicts = client.list_groups()
-        if group_dicts:
-            for row in group_dicts:
-                row['owner'] = '%s(%s)' % (row['owner_name'], row['owner_id'])
-            self.print_table(('name', 'uuid', 'owner', 'role'), group_dicts)
+        client, worksheet_uuid = self.parse_client_worksheet_uuid(args.worksheet_spec, use_rest=True)
+        user_id = client.fetch('user')['id']
+        groups = client.fetch('groups')
+
+        if groups:
+            for group in groups:
+                group['uuid'] = group['id']
+                group['role'] = 'member'
+                for member in group['admins']:
```

Comment:
if any([member['id'] == user_id for member in group['admins']:
  group['role'] = 'admin'